Modules/_struct.c

```diff
@@ -878,13 +853,8 @@ static const formatdef native_table[] = {
     {'e',       sizeof(short),  _Alignof(short),    nu_halffloat,   np_halffloat},
     {'f',       sizeof(float),  _Alignof(float),    nu_float,       np_float},
     {'d',       sizeof(double), _Alignof(double),   nu_double,      np_double},
-#ifdef Py_HAVE_C_COMPLEX
-    {'F',       sizeof(float complex), _Alignof(float complex), nu_float_complex, np_float_complex},
-    {'D',       sizeof(double complex), _Alignof(double complex), nu_double_complex, np_double_complex},
-#else
-    {'F',       1, 0, nu_complex_stub, np_complex_stub},
-    {'D',       1, 0, nu_complex_stub, np_complex_stub},
-#endif
+    {'F',       2*sizeof(float), _Alignof(float[2]), nu_float_complex, np_float_complex},
```

Comment:
Something we should actually care in the future but `_Alignof` is deprecated since C23 and it's renamed as `alignof`. It's unlikely it will be removed but for the future, we should likely use a macro `Py_ALIGNOF` or something like that. It's a follow-up work though.

Reply:
AFAIK, C23 calls it an "Alternative Spelling". I wouldn't worry about it.